commands/command_status.go

```diff
@@ -224,6 +229,39 @@ func statusScanRefRange(ref *git.Ref) {
 
 }
 
+type JsonStatus struct {
+	Files map[string]interface{}
```

Comment:
Do you think it would be worthwhile to make this more type-safe by adding a `JsonStatusEntry` type? For instance:

```go
type JSONStatusEntry struct {
        Status string `json:"status"`
        From string `json:"from"`
}

type JSONStatus struct {
        Files map[string]JSONStatusEntry `json:"files"`
}
```

You can also use the backticks `json:""` after a struct field to apply a custom name to it during formatting, for instance:

```go
type T struct {
        X string `json:"x"`
}
```

would be encoded as `{"x":"..."}` instead of `{"X":"..."}`.